Wireframe/css/style.css

```diff
@@ -0,0 +1,141 @@
+:root {
+  --font-family: sans-serif;
+  --orange-dark: #ef6419;
+  --aqua-color: aqua;
+  --grey-color: grey;
```

Comment:
I think one of the major benefits of variables is that you can add more semantic value to your code. You can describe the behaviour of your value, not just the value. What are the variables giving you here?